$OpenBSD: patch-tmac_an-old_tmac,v 1.3 2013/07/13 07:39:00 schwarze Exp $
chunk 1: Keep traditional OpenBSD page headers (local change).
chunk 2: Correct right margin after double .TP (submitted upstream).
--- tmac/an-old.tmac.orig	Thu Feb  7 13:06:08 2013
+++ tmac/an-old.tmac	Mon Jul  8 07:42:25 2013
@@ -145,18 +145,17 @@
 .    el                  .ds an-extra2 \"Sun Release 4.0
 .
 .    ie (\\n[.$] > 4)    .ds an-extra3 "\\$5\"
-.    el \{ .ie '\\$2'1'  .ds an-extra3 General Commands Manual
-.    el \{ .ie '\\$2'2'  .ds an-extra3 System Calls Manual
-.    el \{ .ie '\\$2'3'  .ds an-extra3 Library Functions Manual
-.    el \{ .ie '\\$2'3p' .ds an-extra3 Perl Programmers Reference Guide
-.    el \{ .ie '\\$2'4'  .ds an-extra3 Kernel Interfaces Manual
-.    el \{ .ie '\\$2'5'  .ds an-extra3 File Formats Manual
-.    el \{ .ie '\\$2'6'  .ds an-extra3 Games Manual
-.    el \{ .ie '\\$2'7'  .ds an-extra3 Miscellaneous Information Manual
-.    el \{ .ie '\\$2'8'  .ds an-extra3 System Manager's Manual
-.    el \{ .ie '\\$2'9'  .ds an-extra3 Kernel Developer's Manual
+.    el .ie '\\$2'1'     .ds an-extra3 OpenBSD Reference Manual
+.    el .ie '\\$2'2'     .ds an-extra3 OpenBSD Programmer's Manual
+.    el .ie '\\$2'3'     .ds an-extra3 OpenBSD Programmer's Manual
+.    el .ie '\\$2'3p'    .ds an-extra3 Perl Programmers Reference Guide
+.    el .ie '\\$2'4'     .ds an-extra3 OpenBSD Programmer's Manual
+.    el .ie '\\$2'5'     .ds an-extra3 OpenBSD Programmer's Manual
+.    el .ie '\\$2'6'     .ds an-extra3 OpenBSD Reference Manual
+.    el .ie '\\$2'7'     .ds an-extra3 OpenBSD Reference Manual
+.    el .ie '\\$2'8'     .ds an-extra3 OpenBSD System Manager's Manual
+.    el .ie '\\$2'9'     .ds an-extra3 OpenBSD Kernel Manual
 .    el                  .ds an-extra3
-.    \}\}\}\}\}\}\}\}\}\}
 .    ds an-init
 \\..
 .
@@ -406,8 +405,10 @@
 .  if \\n[.$] .nr an-prevailing-indent (n;\\$1)
 .  it 1 an-trap
 .  in 0
-.  ll -\\n[an-margin]u
-.  if !\\n[an-div?] .di an-div
+.  if !\\n[an-div?] \{\
+.    ll -\\n[an-margin]u
+.    di an-div
+.  \}
 .  nr an-div? 1
 ..
 .
